package.json

```diff
@@ -81,5 +103,6 @@
     "setupFilesAfterEnv": [
       "@testing-library/jest-native/extend-expect"
     ]
-  }
+  },
+  "sideEffects": false
```

Comment:
This lets Webpack bundlers know that our code is not doing anything funky like change the Prototype or adding stuff to `window` and aids tree shaking